src/Diff/Internal/FingerprintPatcher.php

```diff
@@ -52,6 +52,11 @@ public function patchFingerprint( Fingerprint $fingerprint, EntityDiff $patch )
 		$this->patchAliases( $fingerprint, $patch->getAliasesDiff() );
 	}
 
+	/**
+	 * @param string[] $termArray
+	 *
+	 * @return TermList
+	 */
 	private function newTermListFromArray( $termArray ) {
```

Comment:
#wat. I forgot to add a type hint o_O

![355zu7](https://cloud.githubusercontent.com/assets/146040/10290084/50889cd6-6ba2-11e5-86cb-bafe84c0956e.jpg)